.travis.yml

```diff
@@ -1,23 +1,25 @@
+os: linux
+dist: xenial
 language: python
 python:
-  - 2.7.15
-dist: trusty
+  - "2.7"
+  - "3.7"
+cache:
+  pip: true
 addons:
   apt:
     sources:
       - google-chrome
     packages:
       - google-chrome-stable
 install:
-  - pip install --upgrade pip
```

Comment:
Relying on virtualenv or venv does not work with Travis. Please stick to what we have.